.github/workflows/release_6_docs.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       run: yarn build
 
     - name: (Release - Docs 6.0) Build Website
-      run: yarn antora ./antora-playbook-staging.yml
+      run: yarn antora ./antora-playbook-release.yml
+      env:
+        DOCS_ENV: staging
```

Comment:
Should this be `staging` as this is for production?

Reply:
Butter fingers

Reply:
I changed it but forgot to push, just doing it right now.